content/setup/proxy_reverse.md

```diff
@@ -45,3 +45,41 @@ location / {
     chunked_transfer_encoding off;
 }
 ```
+
+# Apache
+
+Example apache reverse proxy configuration:
+
+```
+<VirtualHost *:80>
+    ServerName drone.mycompany.com
+    Redirect permanent / https://drone.mycompany.com/
+</VirtualHost>
+
+<VirtualHost *:443>
+    ServerName drone.mycompany.com
+
+    SSLEngine on
+    SSLCertificateFile /etc/letsencrypt/live/drone.mycompany.com/fullchain.pem
```

Comment:
I believe so, pretty sure it wont work on mobile phones unless this is fullchain, because I ran into that issue before